Cargo.toml

```diff
@@ -44,6 +45,7 @@ tonic-build = { version = "0.11.0", default-features = false }
 env_logger = "0.11"
 log = "0.4"
 base64 = "0.22"
+sp1-bn254-poseidon = {git = "https://github.com/S1nus/sp1-poseidon2-bn254-scalar"}
```

Comment:
probably we should publish this on crates & version it before merging this PR? :shrug: at least tag a version?